### PR TITLE
Document asset cleanup

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/40223

WIP because I want to read it again myself tomorrow :-)

### CODE REVIEW

File: docs/UsersGuide.asciidoc [hunk 1]
```diff
@@ -271,6 +271,55 @@ provide a rename target, for example
 ISO_1_DECOMPRESS_URL=http://host/foo2.iso.xz and ISO_1=foo.iso
 
 
+== Asset cleanup ==
```
Comment (okurz) [inline docs/UsersGuide.asciidoc]: Looks like you missed the introduction of the section https://github.com/os-autoinst/openQA/pull/1765/files#diff-15b5771a4f9c33cc7735eca26e0b1484R242 recently?

File: docs/UsersGuide.asciidoc [hunk 2]
```diff
@@ -271,6 +271,55 @@ provide a rename target, for example
...
+
+Since assets consume a huge amount of disk space, openQA removes assets
+automatically according to configurable limits.
+
+=== Cleanup stragegy
```
Comment (okurz) [inline docs/UsersGuide.asciidoc]: s/stragegy/strategy/

File: docs/UsersGuide.asciidoc [hunk 3]
```diff
@@ -271,6 +271,55 @@ provide a rename target, for example
...
+If the size limit for assets by a certain group is exceeded, openQA will
+delete those assets:
+
+* Assets belonging to old jobs a preferred.
+* Assets belonging to jobs which are still scheduled or running will not be
+  considered at all.
+* Assets which also belong to another group which still has space left are
+  not considered as well.
+
+Assets which don't belong to any group are deleted after a configurable
```
Comment (okurz) [inline docs/UsersGuide.asciidoc]: I think it is more common to use the full form of words for documentation, s/don't/do not/

File: docs/UsersGuide.asciidoc [hunk 4]
```diff
@@ -271,6 +271,55 @@ provide a rename target, for example
...
+assets which belong to that group and not to any other group.
+
+Job groups inherit the size limit from their parent group unless the limit
+is set explicitely. The default size limit for groups can be adjusted in
+the +default_group_limits+ section of the openQA config file.
+
+=== Configure limit for groupless assets
+
+Assets only belonging to jobs not within a group are deleted automatically
+after 14 days. That duration can be adjusted by setting
```
Comment (okurz) [inline docs/UsersGuide.asciidoc]: I would be a bit more careful by restarting the default values, you know, code changes, documentation becomes outdated …

File: docs/UsersGuide.asciidoc (hunk 1 above)
Comment (Martchus) [reply docs/UsersGuide.asciidoc]: It was under 'API' and I've just read the first lines. So I assumed it is only about the relevant API parameter.

Maybe it is better to have an extra section for it and have only API-specific documentation under the API section.

File: docs/UsersGuide.asciidoc (hunk 3 above)
Comment (Martchus) [reply docs/UsersGuide.asciidoc]: I know, I even used a 'will not' somewhere (even though won't is so much cooler). That one I've just overlooked.

File: docs/UsersGuide.asciidoc (hunk 1 above)
Comment (okurz) [reply docs/UsersGuide.asciidoc]: sure, play around with it, move it and combine what you want to add :)

File: docs/UsersGuide.asciidoc [hunk 5]
```diff
@@ -245,12 +300,7 @@ Multiple parameters exist to reference assets to be used by tests, for example
 `ISO`, `HDD`, `ASSET`, `KERNEL`, `INITRD`. Corresponding files must be
 provided within the path +/var/lib/openqa/share/factory+ local to the openQA
 web-UI. All assets specified in this way in any job are tracked by openQA and
-removed from the filesystem as soon as the corresponding job group disk usage
-quota is depleted and no scheduled or running jobs are relying on the assets.
-Therefore it is useful for production instances to make sure that all provided
-assets are tracked this way to prevent disk space depletion. Still, other
-assets found but not mentioned by any jobs are deleted after a fixed, longer
-time.
+considered by the automatic cleanup described under 'Asset cleanup'.
```
Comment (okurz) [inline docs/UsersGuide.asciidoc]: :+1:

**Pull request merged**